rust_snuba/src/processors/mod.rs

```diff
@@ -55,6 +56,7 @@ define_processing_functions! {
     ("QuerylogProcessor", "snuba-queries", ProcessingFunctionType::ProcessingFunction(querylog::process_message)),
     ("ReplaysProcessor", "ingest-replay-events", ProcessingFunctionType::ProcessingFunction(replays::process_message)),
     ("SpansMessageProcessor", "snuba-spans", ProcessingFunctionType::ProcessingFunction(spans::process_message)),
+    ("SpansV2MessageProcessor", "snuba-spans", ProcessingFunctionType::ProcessingFunction(spans_v2::process_message)),
```

Comment:
Is it fine if this has the same "key" as spans processor? It's unclear if it's just the kafka topic name - these two should have the same topic but different consumer groups

Reply:
This is fine, check `snuba-generic-metrics`, it's basically the same setup.